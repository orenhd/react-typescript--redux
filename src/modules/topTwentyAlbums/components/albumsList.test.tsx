import React from 'react';
import { shallow } from 'enzyme';

import AlbumsList from './albumsList';

/* Import Mocks */
import albumEntriesListMock from '../__mocks__/albumEntriesList.mock.json';

describe('albumsList', () => {
  it('should render correctly', () => {

    const component = shallow(<AlbumsList
      albumEntriesList={albumEntriesListMock}
    />);

    expect(component).toMatchSnapshot();
  });
});
