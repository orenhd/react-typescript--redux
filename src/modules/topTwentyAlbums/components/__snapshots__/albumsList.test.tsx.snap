// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`albumsList should render correctly 1`] = `
<List>
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/walls/1436895674?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is1-ssl.mzstatic.com/image/thumb/Music118/v4/9a/3c/af/9a3caf3e-8c2a-56d9-e13d-f7577d97a475/886447269398.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="Walls"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Barbra Streisand
        </span>
        <br />
        <span>
          ℗ 2018 Columbia Records, a Division of Sony Music Entertainment
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/yes-or-yes/1440972731?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is2-ssl.mzstatic.com/image/thumb/Music118/v4/24/d5/fb/24d5fb61-6d50-e6e8-331e-275be8e2bdad/20181105_6PM_TWICE_6_MAIN_COVER_IMAGE.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="YES or YES"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          TWICE
        </span>
        <br />
        <span>
          ℗ 2018 JYP Entertainment
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/dont-mess-up-my-tempo-the-5th-album/1440782229?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is3-ssl.mzstatic.com/image/thumb/Music128/v4/97/f7/95/97f795bb-178f-53c6-8c4e-f1fa0f559846/EXO_DIGITAL_COVER.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="DON’T MESS UP MY TEMPO – The 5th Album"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          EXO
        </span>
        <br />
        <span>
          ℗ 2018 SM Entertainment
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/now-thats-what-i-call-music-vol-68/1437684126?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is5-ssl.mzstatic.com/image/thumb/Music118/v4/b4/da/47/b4da47e4-1354-1c2e-1a85-c94b2c65893a/00600753851715.rgb.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="NOW That's What I Call Music!, Vol. 68"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Various Artists
        </span>
        <br />
        <span>
          This Compilation ℗ 2018 UMG Recordings, Inc. and Sony Music Entertainment
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/photographs-memories-his-greatest-hits/324594793?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is4-ssl.mzstatic.com/image/thumb/Music/1a/8c/78/mzi.ekdqcsqn.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="Photographs & Memories: His Greatest Hits"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Jim Croce
        </span>
        <br />
        <span>
          ℗ 1974 R2M Music. Marketed by Rhino Entertainment Company, a Warner Music Group Company.
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/deluxe/1193701079?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is2-ssl.mzstatic.com/image/thumb/Music122/v4/9a/35/95/9a3595c7-c675-fd74-352f-44e3638ac123/190295851286.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="÷ (Deluxe)"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Ed Sheeran
        </span>
        <br />
        <span>
          ℗ 2017 Asylum Records UK, a division of Atlantic Records UK, a Warner Music Group company.
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/honey/1436781118?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is2-ssl.mzstatic.com/image/thumb/Music128/v4/67/11/9d/67119d14-d26e-a98e-e0a9-7c705d6ddd44/00602567989752.rgb.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="Honey"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Robyn
        </span>
        <br />
        <span>
          ℗ 2018 Konichiwa Records, under exclusive license to Interscope Records
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/namanana-the-3rd-album/1439328418?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is2-ssl.mzstatic.com/image/thumb/Music128/v4/63/10/d2/6310d215-3307-991f-b824-b91091b446e4/cover.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="NAMANANA - The 3rd Album"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          LAY
        </span>
        <br />
        <span>
          ℗ 2018  SM Entertainment, ZHANG YIXING STUDIO.
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/mono/1439732979?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is5-ssl.mzstatic.com/image/thumb/Music128/v4/48/c3/19/48c31915-36ed-aedc-00c3-d59a0c1f9848/193483116426.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="mono."
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          RM
        </span>
        <br />
        <span>
          ℗ 2018 Bighit Entertainment
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/one-shot/1437841022?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is2-ssl.mzstatic.com/image/thumb/Music128/v4/99/74/38/9974383c-390b-e7c6-6f3b-4613ca26f1c9/075679864321.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="One Shot"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Straight No Chaser
        </span>
        <br />
        <span>
          ℗ 2018 Atlantic Recording Corporation for the United States and WEA International Inc. for the world outside of the United States. A Warner Music Group Company
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/gold-greatest-hits/850009855?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is5-ssl.mzstatic.com/image/thumb/Music4/v4/c5/37/99/c53799cb-9086-98b5-9ff9-85dab96e9111/UMG_cvrart_00602537784608_01_RGB72_1500x1500_14UMGIM07615.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="Gold: Greatest Hits"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          ABBA
        </span>
        <br />
        <span>
          ℗ 2014 Polar Music International AB
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/el-mal-querer/1436309944?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is1-ssl.mzstatic.com/image/thumb/Music118/v4/0f/61/1b/0f611bad-797b-2c33-8dbf-a2e7691e61a0/886447217139.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="EL MAL QUERER"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          ROSALÍA
        </span>
        <br />
        <span>
          ℗ 2018 Sony Music Entertainment España, S.L.
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/b-a-p-concert-special-solo-the-recollection-ep/1441270559?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is3-ssl.mzstatic.com/image/thumb/Music118/v4/0d/47/85/0d4785f0-4967-5eec-68a7-f15e40327182/cover-B.A.P_new.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="B.A.P Concert Special Solo 'the Recollection' - EP"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          B.A.P
        </span>
        <br />
        <span>
          ℗ 2018 티에스이엔티이알(TS ENTER),under license to Loen Entertainment Inc
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/diamonds/1290355394?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is5-ssl.mzstatic.com/image/thumb/Music118/v4/fe/99/2d/fe992de9-482e-ae64-4fec-b06b6ff95a1c/UMG_cvrart_00602557998368_01_RGB72_1800x1800_17UM1IM23523.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="Diamonds"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Elton John
        </span>
        <br />
        <span>
          A Virgin EMI Records Release;℗2017 Universal Music Operations Limited
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/reputation/1274999981?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is3-ssl.mzstatic.com/image/thumb/Music118/v4/ae/8f/46/ae8f4677-0576-a1e4-e360-1563602c9bfb/UMG_cvrart_00843930033133_01_RGB72_3000x3000_17UM1IM24651.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="reputation"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Taylor Swift
        </span>
        <br />
        <span>
          ℗ 2017 Big Machine Label Group, LLC
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/the-essential-michael-jackson/159292399?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is1-ssl.mzstatic.com/image/thumb/Music127/v4/85/c7/e8/85c7e830-3510-aa34-5bbe-a09cb5b1ddc5/827969428726.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="The Essential Michael Jackson"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Michael Jackson
        </span>
        <br />
        <span>
          ℗ 1972 Motown Records, a Division of UMG Recordings, Inc., 1976, 1978, 1980 Sony Music Entertainment, 1979, 1982, 1987, 1991, 1995, 2001, 2005 MJJ Productions Inc.
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/bridges-deluxe/1403975181?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is4-ssl.mzstatic.com/image/thumb/Music125/v4/40/82/ba/4082bad2-8bd4-20ab-3830-153d24e9cfa3/093624904458.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="Bridges (Deluxe)"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Josh Groban
        </span>
        <br />
        <span>
          ℗ 2018 Reprise Records
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/act-5-new-action-ep/1441289791?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is4-ssl.mzstatic.com/image/thumb/Music118/v4/ee/c4/cc/eec4cc79-8449-4f15-46a7-adad801212ec/asfasfafsd.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="ACT.5 New Action - EP"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          gugudan
        </span>
        <br />
        <span>
          ℗ Jellyfish Entertainment
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/shawn-mendes/1376115721?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is5-ssl.mzstatic.com/image/thumb/Music128/v4/c8/ee/e9/c8eee994-a95c-3ddf-1307-4929d52c540b/00602567569466.rgb.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="Shawn Mendes"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Shawn Mendes
        </span>
        <br />
        <span>
          ℗ 2018 Island Records, a division of UMG Recordings, Inc.
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
  <ListItem
    autoGenerateNestedIndicator={true}
    containerElement="span"
    disableKeyboardFocus={false}
    disabled={false}
    initiallyOpen={false}
    insetChildren={false}
    key="https://itunes.apple.com/us/album/dancing-queen/1421985829?uo=2"
    leftAvatar={
      <Avatar
        size={40}
        src="https://is2-ssl.mzstatic.com/image/thumb/Music118/v4/0d/93/50/0d935081-6492-cb68-eca1-93a70e79e8ff/093624904427.jpg/55x55bb-85.png"
      />
    }
    nestedItems={Array []}
    nestedLevel={0}
    onKeyboardFocus={[Function]}
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
    onNestedListToggle={[Function]}
    onTouchEnd={[Function]}
    onTouchStart={[Function]}
    open={null}
    primaryText="Dancing Queen"
    primaryTogglesNestedList={false}
    secondaryText={
      <p>
        <span>
          Cher
        </span>
        <br />
        <span>
          ℗ 2018 Warner Bros. Records Inc.
        </span>
      </p>
    }
    secondaryTextLines={2}
    style={
      Object {
        "pointerEvents": "none",
      }
    }
  />
</List>
`;
